birminghamio.ini

```diff
@@ -237,3 +237,8 @@ glyphicon = briefcase
 name = Stickee
 twitter = stickeeuk
 glyphicon = briefcase
+
+[https://medium.com/initialcommit/]
```

Comment:
This URL needs to be directly to an RSS or Atom feed, not to the front page of the site itself. I'm not sure where they are on Medium; if you can't work it out, let me know and I'll look into it.

Reply:
Is this useful? https://help.medium.com/hc/en-us/articles/214874118-RSS-feeds